testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -246,6 +247,18 @@ protected void checkRun(String path) throws Exception {
               int thresholdValue = ((BigDecimal) value).intValue();
               closer.add(Prepare.THREAD_INSUBQUERY_THRESHOLD.push(thresholdValue));
             }
+            if (propertyName.equals("trimfields")) {
```

Comment:
what I don't understand is "at which point during compilation is field trimming executed"?
The result will depend on the optimizations that are executed before and after.